### DISCUSSION TITLE
Salvar arquivo com o Postman

### DISCUSSION THREAD

**Discussion by RafaelFCouto** [balta-io/forum#439] [Tira Dúvidas] [answered]
Este é o comando que estou mandando no Post do Postman:
{
    "id":1,
    "title":"Ir ao supermercado",
    "done": true,
    "createdAt": "2023-10-10T14:00:00"
    
}
Mas, só está salvando assim: 
{"id":1,"title":"Ir ao supermercado","done":true,"createAt":"0001-01-01T00:00:00"}

Analisando o meu código só identifiquei que criei um atributo na classe TodoModel com essa diferença: 

Código do Balta: Public DateTime CreatedAt
Meu Código: Public DateTime CreateAt

Alguém pode ajudar?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @RafaelFCouto, como vai?
Conforme eu explico nas aulas onde falo sobre Model Binding, o ASP.NET tenta mapear as propriedades do JSON automaticamente para sua classe.

Se no JSON está CREATED, na sua classe tem que estar CREATED também. Por isso deu erro.
Se manter o mesmo nome da classe que mostro no curso vai dar certo (Ou mudar o nome no JSON).
